### ISSUE TITLE
VLLM GRPO推理参数设置出错

### ISSUE THREAD

**Issue by yyyiron** [modelscope/ms-swift#3300]
**Describe the bug**
What the bug is, and how to reproduce, better with screenshots(描述bug以及复现过程，最好有截图)
GRPO官方VLLM示例跑通，但是调整参数后一直显示显存溢出,并且分配给她四张卡，但仅调用0，7显卡

以下为运行脚本:
CUDA_VISIBLE_DEVICES=0,1,6,7 \
NPROC_PER_NODE=3 \
swift rlhf \
    --rlhf_type grpo \
    --model /deepseek-model-5/models/deepseek-ai/DeepSeek-R1-Distill-Qwen-7B  \
    --reward_funcs accuracy format \
    --use_vllm true \
    --vllm_device auto \
    --vllm_gpu_memory_utilization 0.7 \
    --vllm_max_model_len 8192 \
    --train_type full \
    --torch_dtype bfloat16 \
    --dataset 'AI-MO/NuminaMath-TIR#5000' \
    --max_completion_length 2048 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 3 \
    --per_device_eval_batch_size 3 \
    --learning_rate 1e-6 \
    --gradient_accumulation_steps 2 \
    --eval_steps 20 \
    --save_steps 20 \
    --save_total_limit 2 \
    --logging_steps 5 \
    --max_length 4096 \
    --output_dir /deepseek-model-5/output \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --dataset_num_proc 4 \
    --num_generations 3 \
    --temperature 0.9 \
    --max_steps 100 \
    --deepspeed zero2 \
    --log_completions true \
    --report_to wandb \

![Image](https://github.com/user-attachments/assets/ae8e104a-f16f-4db4-979f-37d5600e8749)
报错信息：

![Image](https://github.com/user-attachments/assets/99428c76-87e9-44ef-b00f-3c5147c3ce76)

**Your hardware and system info**
Write your system info like CUDA version/system/GPU/torch version here(在这里给出硬件信息和系统信息，如CUDA版本，系统，GPU型号和torch版本等)
CUDA：12.04
系统:Ubuntu 22.04
GPU:H20
torch:2.5.1
vllm:0.7.2
trl:0.16.0
ms_swift:3.2.0.dev0

**Additional context**
Add any other context about the problem here(在这里补充其他信息)

**Comment by tastelikefeet** [comment]
整体少了反斜杠
```shell
CUDA_VISIBLE_DEVICES=0,1,6,7 \
NPROC_PER_NODE=3 \
swift rlhf \
...
```
